### PR TITLE
[core] Security fix : SA-CORE-2016-002

### PR DESCRIPTION
**Pull request by gboddin**
Urgent, merge after tests passed

### CODE REVIEW

**Comment by gervasek** [comment]
:-1: 

It should be fixed on 2.1-support

**Comment by Fefaine** [comment]
:-1:

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -616,7 +616,7 @@ projects[video][patch][] = patches/video-security-883.patch
 ; See https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-7788
 ; Todo: revert back to the stable version when Views 7.x-3.14 is released.
 projects[views][download][branch] = 7.x-3.x
-projects[views][download][revision] = e5c8b65
```
Comment (gervasek) [inline resources/multisite_drupal_standard.make]: It should be fixed on a separate ticket. A JIRA number should be mentioned on the commit message.

Comment (gervasek) [inline resources/multisite_drupal_standard.make]: I thinks we can point to the version:
```
projects[views][version][branch] = 7.x-3.14
```

**Comment by gboddin** [comment]
Be my guess, awereness is raised ;)

**Pull request closed without merging**